indico/modules/rb_new/controllers/backend/rooms.py

```diff
@@ -106,3 +106,20 @@ def _process_PUT(self):
     def _process_DELETE(self):
         session.user.favorite_rooms.discard(self.room)
         return '', 204
+
+
+# TODO
+class RHRoomUpdate(RHRoomBase):
+    # TODO: Check if current user is allowed to edit room
+    def _check_access(self):
```

Comment:
usually we have the methods in this order since that's also the order in which they are called:

1. `_process_args`
2. `_check_access`
3. `_process` (and anything else)